src/ipc/ipc3/dai.c

```diff
@@ -73,6 +73,9 @@ int dai_config_dma_channel(struct dai_data *dd, struct comp_dev *dev, const void
 	case SOF_DAI_IMX_MICFIL:
 		channel = dai_get_handshake(dd->dai, dai->direction,
 					    dd->stream_id);
+#ifdef CONFIG_ZEPHYR_NATIVE_DRIVERS
```

Comment:
Would be nice to have an inline comment here stating why this is needed.

Reply:
sure.